_posts/patterns/2015-12-21-lists.md

```diff
@@ -0,0 +1,308 @@
+---
+
+<h3 class="hd-6 example-set-hd">List (using different semantics)</h3>
+<div class="example-set">
+    <ul class="list">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ul>
+
+    <ol class="list">
+        <li class="item">Blueberries</li>
+        <li class="item">Cherries</li>
+        <li class="item">Coconuts</li>
+        <li class="item">Cranberries</li>
+    </ol>
+
+    <div class="list">
+        <p class="item">Figs</p>
+        <p class="item">Oranges</p>
+        <p class="item">Plums</p>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Lists with specific sizes</h3>
+<div class="example-set">
+    <ul class="list list-xs">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ul>
+
+    <ul class="list list-sm">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ul>
+
+    <ol class="list list-md">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ol>
+
+    <ol class="list list-lg">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ol>
+
+    <div class="list list-xl">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Inline List</h3>
+<div class="example-set">
+    <ul class="list list-inline">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ul>
+</div>
+
+<h3 class="hd-6 example-set-hd">List with alignments</h3>
+<div class="example-set">
+    <ul class="list">
+        <li class="item align-left">Apples (left aligned)</li>
```

Comment:
Should the alignment be set on the parent? I can't picture a list that would have different alignments set on different items...